tooling/browserify/dot-browser.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+/* eslint-disable no-invalid-this */
+
+const debug = require(`debug`)(`tooling:dot-browser`);
+const fs = require(`fs`);
+const path = require(`path`);
+const {wrap} = require(`lodash`);
```

Comment:
would it be better to adapt the `const wrap = require(`lodash/wrap`);` syntax here as well just to be consistent with the code that supports tree shaking?